.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+install:
+  - docker build -t quay.io/${TRAVIS_REPO_SLUG:0:6}/${TRAVIS_REPO_SLUG:14}:${TRAVIS_COMMIT:0:7} .
```

Comment:
I mostly did this in an attempt to keep the Azavea Docker related Travis setups somewhat consistent.